modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -321,9 +321,11 @@
  * A <b>shearing</b> transformation, sometimes called a skew, effectively
  * rotates one axis so that the x and y axes are no longer perpendicular.
  * <p>
- * Multiple transformations may be applied to a node by specifying an ordered
- * chain of transforms.  The order in which the transforms are applied is
- * defined by the ObservableList specified in the {@link #getTransforms transforms} variable.
+ * Multiple transformations may be applied to a node. Custom transforms are applied first using the {@link #getTransforms transforms}
+ * list. Predefined transforms are applied afterwards in this order: scale, rotation and translation. These are applied using the
+ * {@link #scaleXProperty scaleX}, {@link #scaleYProperty scaleY}, {@link #scaleZProperty scaleZ}, {@link #rotateProperty rotate},
+ * {@link #translateXProperty translateX}, {@link #translateYProperty translateY} and {@link #translateZProperty translateZ}
+ * properties.
```

Comment:
See my comment on the transform property. This section seems like the right place to expand the documentation to specify the order of matrix multiplication and the order in which they affect the node's geometry.